Gemfile

```diff
@@ -6,7 +6,9 @@ eval_gemfile "Gemfile.devtools"
 
 gemspec
 
-gem "dry-configurable"
+# gem "dry-configurable"
+gem "dry-configurable", github: "dry-rb/dry-configurable", branch: "write-settings-in-configure-only"
```

Comment:
I assume this is going to be removed before merging this PR in?

Reply:
Yep! What I'm thinking is that I'll merge the dry-configurable PR first (when it and we are fully ready), then update all these PRs to rely on the main branch before merging them.

Then we'll handle releases after that.